src/main/webapp/index.html

```diff
@@ -47,6 +47,17 @@
       <input type="radio" id="terrible" name="rating-class" value="terrible" />
       <label for="terrible">Terrible.</label>
 
+      <p>How many hours per week did you spend on this class?</p>
+      <input type="number" id="hoursOfWork" name="hoursOfWork" min="1" />
+      <label for="hoursOfWork">Hours (include in-class time)</label>
+
+      <br /><br />
+      <p>How difficult did you find this class?</p>
+      <input type="number" id="difficulty" name="difficulty" min="0" max="5" />
+      <label for="difficulty"
+        >Difficulty (From 0 = Very Easy to 5 = Very Hard)</label
+      >
```

Comment:
I feel like this styling is a little bit weird with the >'s on different lines from their preceding tags?

Reply:
I agree but for some reason prettier is formatting it like that. Will bring it up on the standup.

Reply:
Very odd... I have no idea > . <

Html will strip whitespace padding though so if you put the text inside the label on its own line that might work better and not get mutilated by prettier?